docs/src/shared/readmes/upgrade-guides/v1.1.0.md

```diff
@@ -41,6 +41,11 @@ a prop.
 > The Drawer/NavigationDrawer will still render the overlay for temporary drawers in a new subtree. If the Drawer
 starts out as `defaultVisible` from the server, there _might_ be a server side rendering error message still.
 
```

Comment:
Add documentation about how to migrate with the new menu props and the other changes of `isOpen` -> `visible`, `defaultOpen` -> `defaultVisible`, `onMenuToggle` -> `onVisibilityChange`